poetry.toml

```diff
@@ -1,2 +0,0 @@
-[virtualenvs]
```

Comment:
Does it works like this ? We prefer to use :
`poetry config virtualenvs.in_project true` (this command typo may be wrong)

Reply:
I don't really know, but it does work fine if I either run `poetry run translate` or if I run the code in vscode.